src/sorting.cpp

```diff
@@ -1,13 +1,12 @@
 /**
- * @file braces.cpp
- * Adds or removes braces.
```

Comment:
This would need a new description. Something like: "Sorts chunks and imports" maybe?